terraform/testcases/ecshealthcheck/otconfig.tpl

```diff
@@ -0,0 +1,25 @@
+extensions:
+  health_check:
+    endpoint: "localhost:13133"
+    path: "/health/status"
+receivers:
+  prometheus:
+    config:
+      global:
+        scrape_interval: 15s
+      scrape_configs:
+      - job_name: "test-prometheus-sample-app"
+        static_configs:
+        - targets: [ ${sample_app_listen_address_host}:${sample_app_listen_address_port} ]
+exporters:
+  logging:
+      loglevel: debug
```

Comment:
Does this need to use the log level variable also?

Reply:
In my opinion, it's not required here. for ref: https://github.com/aws-observability/aws-otel-test-framework/blob/812a4eef108a878c28f76f8493b4e1b38f590370/terraform/testcases/otlp_metric_mock/otconfig.tpl